projects/aca-content/src/lib/components/library-list/library-list.component.html

```diff
@@ -0,0 +1,94 @@
+        [node]="$any(list)"
+        [loading]="isLoading"
+        [selectionMode]="'multiple'"
+        [multiselect]="true"
+        [navigate]="false"
+        [sorting]="['title', 'asc']"
+        [sortingMode]="'client'"
+        [displayCheckboxesOnHover]="true"
+        [preselectNodes]="selectedNodesState?.nodes"
+        (node-dblclick)="handleNodeClick($event)"
+        [imageResolver]="imageResolver"
+        (selectedItemsCountChanged)="onSelectedItemsCountChanged($event)"
+        [isResizingEnabled]="true"
+        [blurOnResize]="false"
+        (name-click)="handleNodeClick($event)"
+      >
+        <adf-custom-empty-content-template>
+          <adf-empty-content
+            icon="library_books"
+            [title]="'APP.BROWSE.LIBRARIES.EMPTY_STATE.ALL_LIBRARIES.TITLE'"
+            subtitle="APP.BROWSE.LIBRARIES.EMPTY_STATE.ALL_LIBRARIES.TEXT"
+          />
+        </adf-custom-empty-content-template>
+
+        <data-columns>
+          <ng-container *ngFor="let column of columns; trackBy: trackByColumnId">
+            <ng-container *ngIf="column.template && !(column.desktopOnly && isSmallScreen)">
+              <data-column
+                [id]="column.id"
+                [draggable]="column.draggable"
+                [resizable]="column.resizable"
+                [key]="column.key"
+                [title]="column.title"
+                [type]="column.type"
+                [format]="column.format"
+                [class]="column.class"
+                [sortable]="column.sortable"
+                [isHidden]="column.isHidden"
+                [sortingKey]="column.sortingKey || column.key"
+              >
+                <ng-template let-context>
+                  <adf-dynamic-column [id]="column.template" [context]="context" />
+                </ng-template>
+              </data-column>
+            </ng-container>
+
+            <ng-container *ngIf="!column.template && !(column.desktopOnly && isSmallScreen)">
+              <data-column
+                [id]="column.id"
+                [key]="column.key"
+                [draggable]="column.draggable"
+                [resizable]="column.resizable"
+                [title]="column.title"
+                [type]="column.type"
+                [format]="column.format"
+                [class]="column.class"
+                [sortable]="column.sortable"
+                [isHidden]="column.isHidden"
+                [sortingKey]="column.sortingKey || column.key"
+              />
+            </ng-container>
+          </ng-container>
+        </data-columns>
+      </adf-document-list>
+
+      <adf-pagination
+        [target]="documentList"
+        [pagination]="pagination"
+        (changePageSize)="onChangePageSize($event)"
+        (changePageNumber)="onChange($event)"
+        (nextPage)="onChange($event)"
+        (prevPage)="onChange($event)"
+      />
+    </div>
+
+    <div class="aca-sidebar" *ngIf="infoDrawerOpened$ | async">
```

Comment:
This div is redundant - you can move class and *ngIf to set it on aca-info-drawer tag directly